src/middleware.ts

```diff
@@ -0,0 +1,9 @@
+import { authMiddleware } from '@clerk/nextjs';
+
+export default authMiddleware({
+  publicRoutes: ['/'],
+});
+
+export const config = {
+  matcher: ['/((?!.+\\.[\\w]+$|_next).*)', '/', '/(api|trpc)(.*)'],

```

Comment:
what's this doing? i see regex my brain goes blah

Reply:
all routes are protected by Clerk's authentication middleware, with the exception of internal /_next/ routes and static files